src/uwtools/config/support.py

```diff
@@ -66,6 +67,15 @@ def log_and_error(msg: str) -> Exception:
     return UWConfigError(msg)
 
 
+def yaml_to_str(cfg: dict) -> str:
+    """
+    Returns a uwtools-conventional YAML representation of the given dict.
+
+    :param cfg: A dict object.
+    """
+    return yaml.dump(cfg, default_flow_style=False, sort_keys=False, width=math.inf).strip()
```

Comment:
Here's the canonical `yaml.dump()` call, used (hopefully) wherever possible for consistent output -- including, now, un-wrapped lines.